src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
-body {
-  margin: 0;
+@import './utils/variables';
+@import './header';
+@import './card/index';
+
+.cards {
+  display: grid;
+  grid-template-columns: minmax(200px, 200px);
```

Comment:
The `grid-template-columns` property with `minmax(200px, 200px)` is effectively the same as setting a fixed width of `200px`. Consider using a different value for the second parameter to allow for flexibility if needed.